doc/guides/build_tools.rst

```diff
@@ -0,0 +1,107 @@
+.. _build_tools:
+
+Build tools
+###########
+
+Zephyr project uses a collection of build and post-build scripts to
+assist creation of application images, as described throughout the
+Zephyr documentation, and summarized here.
+
```

Comment:
missing IRQ scripts in arch/common

Reply:
Thanks.  @andrewboie Can you add a docstring description to ``arch/common/gen_isr_tables.py``

Reply:
will do